src/TTrigger.cpp

```diff
@@ -171,7 +171,7 @@ bool TTrigger::setRegexCodeList(QStringList regexList, QList<int> propertyList)
 
             int erroffset;
 
-            QSharedPointer<pcre> re(pcre_compile(local8Bit.constData(), 0, &error, &erroffset, 0), pcre_deleter);
+            QSharedPointer<pcre> re(pcre_compile(local8Bit.constData(), 0, &error, &erroffset, nullptr), pcre_deleter);
```

Comment:
... although in the light of feedback from @ahmedcharles  I guess it isn't an issue.